src/index.ts

```diff
@@ -103,6 +104,10 @@ async function run(): Promise<void> {
         registryPath,
     ];
 
+    if (extraArgs) {
+        args.push(extraArgs);
```

Comment:
I think the extra-args may be best off split over newlines so the user doesn't have to quote them.